config/runtime.exs

```diff
@@ -259,6 +296,9 @@ if config_env() == :test do
 
   config :t, T.Feeds.SeenPruner, disabled?: true
   config :t, T.Feeds.LiveModeManager, disabled?: true
+
+  # there is no user with this id, it's been generated just for the tests
+  config :t, oldies: ["0000017d-d720-1f6a-06e9-e8bbc6570000"]
```

Comment:
we can start with the following "oldies":
Vlad: 0000017b-b0cf-d3e8-0242-ac1100020000
Lily: 0000017b-e3d2-2b30-0242-ac1100020000
me: 0000017c-14c7-9745-0242-ac1100020000